agnostic-svelte-ts/src/lib/index.js

```diff
@@ -0,0 +1,70 @@
+import Alert from '$lib/components/Alert/Alert.svelte';
```

Comment:
Will this `index.js` file become `index.ts` at some point?

Reply:
Oops forgot to update the extension when adding it, sorry! As it only has imports and exports, I think we can safely update this any time. 🙂